discourse-mixin/dashboards/discourse-jobs.libsonnet

```diff
@@ -0,0 +1,621 @@
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 's',
+    },
+    overrides: [],
+  },
+  id: 19,
+  links: [],
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+  pluginVersion: '9.1.8',
+  targets: [
+    prometheus.target(
+      'sum(rate(discourse_sidekiq_job_duration_seconds{instance=~"$instance",job=~"$job"}[$__rate_interval])) by (job_name)',
+      datasource=promDatasource,
+      legendFormat='{{job_name}}'
+    ),
+  ],
+  title: 'Sidekiq Job Duration',
+  type: 'timeseries',
+};
+
+local sheduledJobDurationPanel = {
+  datasource: promDatasource,
+  description: 'Time spent in scheduled jobs broken out by job name.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 30,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'normal',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 's',
+    },
+    overrides: [],
+  },
+  id: 20,
+  links: [],
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+  pluginVersion: '9.1.8',
+  targets: [
+    prometheus.target(
+      'sum(rate(discourse_scheduled_job_duration_seconds{instance=~"$instance",job=~"$job"}[$__rate_interval])) by (job_name)',
+      datasource=promDatasource,
+      legendFormat='{{job_name}}',
+    ),
+  ],
+  title: 'Scheduled Job Duration',
+  type: 'timeseries',
+};
+
+local usedRSSMemoryPanel = {
+  datasource: promDatasource,
+  description: 'Total RSS Memory used by process. Broken up by process type.',
```

Comment:
The label that is being aggregated on is `pid`. Is that actually a process type, or is it the standard system PID of a given process?

If it is actually the PID, this description should reflect that.